docs/contributor/04-30-ci-cd.md

```diff
@@ -50,7 +50,7 @@ The following CI jobs are part of the development cycle.
 | [`UI Tests / Run UI tests`](https://github.com/kyma-project/api-gateway/blob/dda6608241107d653e68fec8503ce9489eab57f3/.github/workflows/ui-tests.yaml#L8)                                            | Runs UI tests for Busola extensions of the API Gateway module and APIRules.                                                                                  |
 | [`Workflow validation / Ensure SHA pinned actions`](https://github.com/kyma-project/api-gateway/blob/dda6608241107d653e68fec8503ce9489eab57f3/.github/workflows/verify-commit-pins.yaml#L10)         | Verifies that all external actions are pinned to a commit.                                                                                                   |
 | [`pull-gitleaks / scan`](https://github.com/kyma-project/api-gateway/blob/dda6608241107d653e68fec8503ce9489eab57f3/.github/workflows/pull-gitleaks.yaml#L10)                                         | Runs the Gitleaks scan to detect any leaks of secrets.                                                                                                       |
-
+| [`PR Build and Push Manager Image / Build and push manager image`](https://github.com/kyma-project/api-gateway/blob/8b3eb837f1474c2e35eb592afbc2e86f69b963f4/.github/workflows/pull-build-push.yaml#L44) | Builds the managers's image and pushs it to a GCP Artifact Registry so that the jobs running tests on Gardener clusters can be executed.               |
```

Comment:
```suggestion
| [`PR Build and Push Manager Image / Build and push manager image`](https://github.com/kyma-project/api-gateway/blob/8b3eb837f1474c2e35eb592afbc2e86f69b963f4/.github/workflows/pull-build-push.yaml#L44) | Builds the manager's image and pushes it to a GCP Artifact Registry so that the jobs running tests on Gardener clusters can be executed.               |
```